packages/s/sv2v/package.yml

```diff
@@ -0,0 +1,28 @@
+name       : sv2v
+version    : 0.0.13
+release    : 1
+source     :
+    - https://github.com/zachjs/sv2v/archive/refs/tags/v0.0.13.tar.gz : 4ce7df8c6fa3857da6a2b69343c29e7c627a4283090f2b07221aa9ef956a88c8
+homepage   : https://github.com/zachjs/sv2v
+license    : BSD-3-Clause
+networking : yes
+component  : programming.tools
+summary    : SystemVerilog to Verilog conversion
+description: |
+    sv2v converts SystemVerilog (IEEE 1800-2017) to Verilog (IEEE 1364-2005), with an emphasis on supporting synthesizable language constructs.
+builddeps  :
+    - haskell-cabal-install
+setup      : |
+    # cabal/ghc-pkg won't recognize binaries installed as build-tool-depends, so we have to install them manually
```

Comment:
Hmmm I'm surprised that this is needed. Let me give it a poke later today and see if I can get rid of this.